### ISSUE TITLE
Not In A Guild

### ISSUE THREAD

**Issue by wiscel** [WarEmu/WarBugs#8814]
Since this afternoon all of us from the Nazgûl guild are not in a guild anymore. This morning everything worked fine. Now all of our alts are guildless, please look into this.

**Comment by Soulwhisper** [comment]
You mean the roster visual bug which shows only your player? 1/1? Or you are completely kicked out and the UI shows Search Interface for Guilds?

**Comment by wiscel** [comment]
No, none of us are in a guild anymore, when you check g our guild is not listed in there. Not even the old Nazgul is there. Also I see guilds from order as well in the list.

**Comment by Soulwhisper** [comment]
You talked with some people in game.Its not the visual bug.

**Comment by wiscel** [comment]
No, its not the visual bug We are not in a guild anymore.

**Comment by Kiraliel** [comment]
We have the same problem. Totally dissappeared and I was the guild leader for it so couldn't have got kick or anything. Just vanished. Hope we see some help for this.

Guild name = Fallen Legion
Guild Leader = Thamor

**Comment by Natherul** [comment]
There was an issue with SC handler that required a reboot of the server and hopefully when the server rebooted it just failed to load all guilds. 

We are aware of the issue and looking at it

**Comment by Kiraliel** [comment]
Ty for answer :)

**Comment by Natherul** [comment]
I think this is now resolved for everyone since the patch?

**Comment by wiscel** [comment]
Yes, Nazgûl guild works again, thank you.
